test/validation/api/ipsec/ipsec.c

```diff
@@ -1155,6 +1163,52 @@ int ipsec_config(odp_instance_t ODP_UNUSED inst)
 	ipsec_config.inbound.chksums.all_chksum = ~0;
 	ipsec_config.stats_en = true;
 
+	ipsec_config.inbound.reassembly.max_wait_time = 20 * ODP_TIME_MSEC_IN_NS;
+	ipsec_config.inbound.reassembly.max_num_frags = capa.reassembly.max_num_frags;
```

Comment:
If max_num_frags capa is less than 4, the 4 fragment reassembly tests are going to fail. Maybe it would be good to check the capa here or somewhere and either skip the 4 fragment tests or give up reassembly testing altogether. Even CU_ASSERT or CU_FAIL would appear ok to me (it can be fixed if an implementation with such a small capa value appears).

Reply:
Valid point. I'll keep reassembly tests disabled when capability doesn't match MAX_FRAGS. In order to selectively support, we may need some additional flags